libraries/c_sdk/aws/defender/src/private/aws_iot_defender_internal.h

```diff
@@ -213,6 +213,10 @@
     #define AWS_IOT_DEFENDER_DEFAULT_PERIOD_SECONDS    ( 300 )
 #endif
 
+#ifndef AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP
+    #define AWS_IOT_DEFENDER_DEFAULT_INVALID_METRICS_GROUP    ( 10000 )
```

Comment:
If this is used only for tests and not by the library, then this macro should be defined in `aws_iot_tests_defender_unit.c`